ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -556,6 +560,24 @@ public void shouldRecoverDrop() {
     shouldRecover(commands);
   }
 
+  @Test
+  public void shouldNotDeleteTopicsOnRecovery() {
```

Comment:
Question:

So with the new logic the rest-api deletes the topic and removes the `DELETE TOPIC` from the command. So in this test  `DROP STREAM B DELETE TOPIC` becomes `DROP STREAM B` in the command topic... right?

So what happens if there is _already_ a legacy  `DROP STREAM B DELETE TOPIC` in the command topic?  Will it still try to delete the topic? Is there a test for this?

Reply:
1. yes
2. it will do nothing - I will add a test for this (the backwards incompatible change is intentional, it is unlikely that they intend for it to be re-deleted - especially in conjunction with the change that prevents it from being re-created on recovery anyway)